Installation problem

Hello,

following the installation steps:
```bash
> go get -u github.com/mbrt/gmailctl/cmd/gmailctl
# github.com/mbrt/gmailctl/pkg/errors
go/src/github.com/mbrt/gmailctl/pkg/errors/err.go:13:8: undefined: errors.Is
go/src/github.com/mbrt/gmailctl/pkg/errors/err.go:14:8: undefined: errors.As
go/src/github.com/mbrt/gmailctl/pkg/errors/err.go:41:6: undefined: errors.As
go/src/github.com/mbrt/gmailctl/pkg/errors/err.go:72:9: undefined: errors.Is
go/src/github.com/mbrt/gmailctl/pkg/errors/err.go:76:5: undefined: errors.As
go/src/github.com/mbrt/gmailctl/pkg/errors/err.go:79:9: undefined: errors.As
# github.com/hashicorp/go-multierror
go/src/github.com/hashicorp/go-multierror/multierror.go:112:9: undefined: errors.As
go/src/github.com/hashicorp/go-multierror/multierror.go:117:9: undefined: errors.Is
```

```bash
> go version
go version go1.12.10 linux/amd64
> echo $GOPATH
/home/enrico/go
```

What am I missing?

Thanks

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/mbrt/gmailctl
 
-go 1.12
+go 1.13
 
 require (
 	cloud.google.com/go v0.41.0 // indirect

